admin/partials/page_sections.php

```diff
@@ -0,0 +1,53 @@
+<div class="wrap apple-news-sections">
+	<h1 id="apple_news_sections_title"><?php esc_html_e( 'Manage Sections', 'apple-news' ) ?></h1>
+	<h2><?php esc_html_e( 'Section', 'apple-news' ) ?>/<?php echo esc_html( $taxonomy->labels->singular_name ); ?> <?php esc_html_e( 'Mappings', 'apple-news' ) ?></h2>
+	<p><?php esc_html_e( 'To enable automatic section assignment, choose the', 'apple-news' ); ?>
```

Comment:
This should be one string using a `sprintf` because the variable might not be in the same place in every language depending on syntax.

So like:

```
echo sprintf(
	__( 'To enable automatic section assignment, choose the %s that you would like to be associated with each section.', 'apple-news' ),
	esc_html( strtolower( $taxonomy->label ) )
);
```